src/test/java/org/opensearch/ad/TestHelpers.java

```diff
@@ -636,7 +637,7 @@ public static AnomalyResult randomMutlEntityAnomalyDetectResult(double score, do
             Instant.now().truncatedTo(ChronoUnit.SECONDS),
             Instant.now().truncatedTo(ChronoUnit.SECONDS),
             error,
-            Arrays.asList(new Entity(randomAlphaOfLength(5), randomAlphaOfLength(5))),
+            Entity.createSingleAttributeEntity(randomAlphaOfLength(5), randomAlphaOfLength(5), randomAlphaOfLength(5)),
```

Comment:
Why create a single attribute entity for a function that looks like it should return a multi-entity result?

Reply:
Multi-entity means a detector with at least one category fields.  It is different form multi-attribute entity that means more than one category field. I changed the function name from randomMutlEntityAnomalyDetectResult to randomHCADAnomalyDetectResult to make it clearer.